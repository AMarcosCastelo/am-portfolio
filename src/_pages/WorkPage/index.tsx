import * as S from './styles'

const WorkPage = () => (
  <S.Wrapper>
    <h1>WorkPage</h1>
  </S.Wrapper>
)

export default WorkPage
