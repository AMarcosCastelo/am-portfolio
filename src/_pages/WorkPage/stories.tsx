import { Story, Meta } from '@storybook/react'
import WorkPage from '.'

export default {
  title: 'WorkPage',
  component: WorkPage
} as Meta

export const Default: Story = () => <WorkPage />
