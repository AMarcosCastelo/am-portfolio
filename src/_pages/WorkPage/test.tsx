import { render, screen } from '@testing-library/react'

import WorkPage from '.'

describe('<WorkPage />', () => {
  it('should render the heading', () => {
    const { container } = render(<WorkPage />)

    expect(
      screen.getByRole('heading', { name: /WorkPage/i })
    ).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
