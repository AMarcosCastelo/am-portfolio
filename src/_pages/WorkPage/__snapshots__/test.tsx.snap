// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<WorkPage /> should render the heading 1`] = `
<main
  class=""
>
  <h1>
    WorkPage
  </h1>
</main>
`;
